### PR TITLE
[153] Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by Mayowa2020**
Product page design using css.
Screenshot (159)
Screenshot (160)
Screenshot (161)
Screenshot (163)
Screenshot (164)
Screenshot (165)

### CODE REVIEW

**Comment by Ireti203** [comment]
Weldone @Mayowa2020. Kindly resolve the conflicts on this branch

File: submissions/BukkyOyetimehin/HTML-Forms/product.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		<header>
+		<!-- Heading-->
+			<h1>Men's Aviator Sunglasses - Designer & Fashion Sunglasses<h1>			
+			<h2>TOP MEN'S AVIATOR SUNGLASSES</h2>
+			<hr>
+			<h3>The Sunglasses To Brighten Up Your Style</h3>
+		</header>
+		<nav>
+			<ul>
+				<li><a href="contact.html">Contact Us</a></li>
```
Comment (Ireti203) [inline submissions/BukkyOyetimehin/HTML-Forms/product.html:18]: You should also have the link to the product page here.

**Review by Ireti203 — CHANGES_REQUESTED** [review]
You have two separate files for the product page, index.html and product.html. You should remove one of them. The css should be added to the previously existing html file for the product page. You don't have to create a new one.

**Comment by adaugochi** [comment]
You still have a conflict on your PR. Kindly check the pinned message on #git-versioncontrol channel on slack. You will see a post on how to resolve merge conflict

**Comment by Mayowa2020** [comment]
> You still have a conflict on your PR. Kindly check the pinned message on #git-versioncontrol channel on slack. You will see a post on how to resolve merge conflict

**Comment by Mayowa2020** [comment]
@adaugochi @Ireti203 Based on the comment, kindly confirm if the conflict issue has been resolved.

**Pull request closed without merging**